CHANGELOG.md

```diff
@@ -1,11 +1,27 @@
 # Upcoming
 
+# v8.0.0-beta
+
 ## Breaking changes
 
-- (>=v8.0.0) Setting model data for a `qx.ui.table.Table` when the table is still editing will
+- Setting model data for a `qx.ui.table.Table` when the table is still editing will
 now raise an error as this could have lead to an invalid edit. To prevent any errors, ensure
 that the table edits are completed or cancelled before refreshing table model data.
 
+- Properties and members are now in the same namespace. Formerly, a class could have a member variable and a property with the same name, and there was no conflict. Now, since properties are first-class and can be manipulated as normal members, the properties and members use the same namespace, so a single name can not be defined in both.
```

Comment:
Tested one of my apps and promptly ran into this problem. My class `callbackery.ui.Busy` uses 'show' as a method which collides with the 'show' property of the `qx.ui.basic.Atom` class.

For catching such issues it would be extremely helpful if the compiler could flag them.


```javascript
qx.Class.define("callbackery.ui.Busy", {
    extend : qx.ui.basic.Atom,
    type : "singleton",

    construct : function() {
        this.base(arguments);
        this.set({
            center : true,
            show: 'both',
            iconPosition: 'top',
            visibility: 'excluded',
            zIndex: 10000001
        });
        this.getApplicationRoot().add(this,{top:0,bottom:0,left:0,right:0});
        this.__blocker = new qx.ui.core.Blocker(this.getApplicationRoot()).set({
            opacity: 0.7,
            color   : '#fff',
            keepBlockerActive: true
        });
        this.__blocker.getBlockerElement().setStyle('zIndex',10000000);
    },
    members : {
        __blocker : null,
        show: function(label){
            this.setVisibility('visible');
            this.setLabel(label);
            this.__blocker.block();
        },
        hide: function(){
            this.setVisibility('excluded');
            this.fadeOut(10);
            this.__blocker.unblock();
        }
    }
});
```
All I get now, when using my `Busy` class 

```javascript
let mybusy = callbackery.ui.Busy.getInstance();
mybusy.show('loading data');
```


is an error message in the browser console at runtime with the complaint:

```
Uncaught TypeError: mybusy.show is not a function
```